test/visual_regression/tests/discover/chart_visualization.js

```diff
@@ -52,28 +52,40 @@ export default function ({ getService, getPageObjects }) {
 
       it('should show bars in the correct time zone', async function () {
         await PageObjects.header.awaitGlobalLoadingIndicatorHidden();
-        await visualTesting.snapshot();
+        await PageObjects.discover.waitUntilSearchingHasFinished();
+        await PageObjects.common.sleep(400);
+        await visualTesting.snapshot({
+          show: ['discoverChart'],
+        });
       });
 
       it('should show correct data for chart interval Hourly', async function () {
         await PageObjects.header.awaitGlobalLoadingIndicatorHidden();
+        await PageObjects.discover.waitUntilSearchingHasFinished();
+        await PageObjects.common.sleep(400);
         await PageObjects.discover.setChartInterval('Hourly');
-        await visualTesting.snapshot();
+        await visualTesting.snapshot({
+          show: ['discoverChart'],
+        });
       });
 
       it('should show correct data for chart interval Daily', async function () {
         await PageObjects.header.awaitGlobalLoadingIndicatorHidden();
+        await PageObjects.discover.waitUntilSearchingHasFinished();
+        await PageObjects.common.sleep(400);
         await PageObjects.discover.setChartInterval('Daily');
-        await retry.try(async () => {
-          await visualTesting.snapshot();
+        await visualTesting.snapshot({
+          show: ['discoverChart'],
         });
       });
 
       it('should show correct data for chart interval Weekly', async function () {
         await PageObjects.header.awaitGlobalLoadingIndicatorHidden();
+        await PageObjects.discover.waitUntilSearchingHasFinished();
+        await PageObjects.common.sleep(400);
```

Comment:
What's the reasoning behind these sleeps? Sleeping is really a last resort option, instead we should use `retry.waitFor()` to wait for a condition to be true rather than waiting an arbitrary amount of time.

Reply:
The text assets are sometimes not loading fast enough. I think this is a global issue but I added those just in case to see if the tests are more stable.

Reply:
@nickofthyme can you try running percy exec -t 500 ? It should fix the asset loading problems.  I did a PR: #43990 but visual testing is still disabled on master, so I have not been able to collect data from kibana-ci.  However, I have been running tests external to kibana-ci this way and so far I have not seen any issues related to this.  So, remove the sleeps and try running that way and let me know if it does not resolve the problem.  Thanks.

Reply:
How about we don't worry about the text assets loading fast enough in this PR, merge this into master, and @liza-mae can make sure the new timeouts in #43990 work for this too? This PR also doesn't re-enable visual regression tests.

Reply:
c767047